Samples/Startup.cs

```diff
@@ -77,14 +90,14 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             // - method names map to http verbs
             // - route paramters are automatically bound by name
             // - complex types are automatically [FromBody]
-            app.UseHttpHandler<ProductsApi3>(model =>
+            routes.MapHttpHandler<ProductsApi3>(model =>
             {
                 model.MapMethodNamesToHttpMethods();
                 model.MapRouteParametersToMethodArguments();
                 model.MapComplexTypeArgsToFromBody();
             });
 
-            app.UseHttpHandler<MyHandler>();
+            routes.MapHttpHandler<MyHandler>();
```

Comment:
I don't see any of these examples using auth. Would they look like?
```
routes.MapHttpHander<TMyTHing>(model =>
{
....
})
.AddAuthorization(.,..)
```

What would you do if you want to authorize differently within a handler 😆

Reply:
Ah!